sdk/spring/azure-spring-boot/src/main/java/com/microsoft/azure/spring/autoconfigure/cosmos/CosmosHealthIndicator.java

```diff
@@ -21,32 +21,32 @@
 public class CosmosHealthIndicator extends AbstractHealthIndicator {
 
     private static final Logger LOGGER = LoggerFactory.getLogger(CosmosHealthIndicator.class);
-    @Value("${azure.cosmosdb.database}")
+    @Value("${azure.cosmos.database}")
     private String dbName;
-    @Value("${azure.cosmosdb.uri}")
+    @Value("${azure.cosmos.uri}")
     private String uri;
 
     @Autowired
     private CosmosClient cosmosClient;
 
     public CosmosHealthIndicator(CosmosClient cosmosClient) {
-        super("CosmosDB health check failed");
+        super("Cosmos health check failed");
         Assert.notNull(cosmosClient, "CosmosClient must not be null");
         this.cosmosClient = cosmosClient;
     }
 
     @Override
     protected void doHealthCheck(Builder builder) throws Exception {
-        CosmosDatabaseResponse response = this.cosmosClient.getDatabase(dbName).read().block();
+        CosmosDatabaseResponse response = this.cosmosClient.getDatabase(dbName).read();
 
         if (response != null) {
             LOGGER.info("The health indicator cost {} RUs, cosmos uri: {}, dbName: {}",
-                response.requestCharge(), uri, dbName);
+                response.getRequestCharge(), uri, dbName);
         }
         if (response == null) {
             builder.down();
         } else {
-            builder.up().withDetail("database", response.database().id());
+            builder.up().withDetail("database", response.getProperties().getId());
         }
```

Comment:
Just to double check, is this ID the ID of the database?

Reply:
yes, 
```java
    /**
     * Gets the cosmos database properties
     *
     * @return the cosmos database properties
     */
    public CosmosDatabaseProperties getProperties() {
        return super.getProperties();
    }

    /**
     * Gets the name of the resource.
     *
     * @return the name of the resource.
     */
    public String getId() {
        return this.database.getId();
    }
```